src/pages/ValidateLoginPage/index.website.tsx

```diff
@@ -48,6 +49,10 @@ function ValidateLoginPage({
             return;
         }
 
+        if (!ValidationUtils.isValidValidateCode(validateCode)) {
```

Comment:
This PR caused a bug. More details here - https://github.com/Expensify/App/issues/54174#issuecomment-2545232016